### PR TITLE
Added prerequesties

### PR DESCRIPTION
**Pull request by surjithctly**
Docs: added node and Git prerequesties #8210

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Hey appreciate this! Does windows not come with git installed? I know macs do.

File: www/src/pages/docs/index.js [hunk 1]
```diff
@@ -22,6 +22,12 @@ class IndexRoute extends React.Component {
               Get started
             </h1>
             <p>Gatsby is a blazing fast modern site generator for React.</p>
+           <h2>Prerequisites</h2>
+             <p> <strong> Node.js</strong>: Before Installing Gatsby, make sure you have Node.js and npm package manager installed on your system. If not, go to <a href="https://nodejs.org/" target="_blank">nodejs.org</a> and install a recent version for your operating system. </p>
+             <p> <strong> Git</strong>: Git is also required to setup Gatsby. If your system do not have Git installed, Install it from <a href="https://git-scm.com/downloads" target="_blank"> here</a></p>
```
Comment (LeKoArts) [inline www/src/pages/docs/index.js]: You'd need to add a `rel="noopener noreferrer"` tag there

**Comment by LeKoArts** [comment]
No, Windows doesn't come with Git afaik

File: www/src/pages/docs/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (surjithctly) [reply www/src/pages/docs/index.js]: Fixed

File: docs/tutorial/part-zero/index.md [hunk 2]
```diff
@@ -55,6 +55,12 @@ npm has three distinct components: the npm website, the npm registry, and the np
 
 > 💡 Check out npm’s introduction, “[**What is npm?**](https://docs.npmjs.com/getting-started/what-is-npm)”.
 
+## Install Git
+
+Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency. Gatsby requires Git so that you can easily clone starter templates from Github repo.
```
[the remark this answers is not in the document]
Comment (surjithctly) [reply docs/tutorial/part-zero/index.md]: Fixed

File: docs/tutorial/part-zero/index.md [hunk 3]
```diff
@@ -55,6 +55,12 @@ npm has three distinct components: the npm website, the npm registry, and the np
...
+
+If your system do not have Git installed, Install it from [https://git-scm.com/downloads](https://git-scm.com/downloads)
```
[the remark this answers is not in the document]
Comment (surjithctly) [reply docs/tutorial/part-zero/index.md]: Fixed

File: www/src/pages/docs/index.js (hunk 1 above)
Comment (surjithctly) [reply www/src/pages/docs/index.js]: Added.

**Comment by surjithctly** [comment]
@KyleAMathews Yeah. Windows does not come with Git by default. need to install it separately. This made a big confusion for me since I was not able to get up and running with gatsby even I followed all steps.

**Comment by KyleAMathews** [comment]
Could you link to part 0 of the tutorial from the quick start? The quick start is intended for more experienced developers so we assume they do have git and node installed. 

Maybe after the first paragraph say: "This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, head to our tutorial!"

**Comment by surjithctly** [comment]
I have this line after prerequesties. 

May be I will remove it and add your text after the first para. Is that okay?

**Comment by surjithctly** [comment]
Ok. Updated 👍

**Comment by thebiltheory** [comment]
> Could you link to part 0 of the tutorial from the quick start? The quick start is intended for more experienced developers so we assume they do have git and node installed.
> 
> Maybe after the first paragraph say: "This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, head to our tutorial!"

Also maybe give them the `Brew` option since they will need to install Node & Git? 
- [https://brew.sh/](Brew.sh)

**Comment by m-allanson** [comment]
> Also maybe give them the Brew option since they will need to install Node & Git?

@thebiltheory 👍 for brew, but unfortunately it's Mac specific - introducing package managers for different OSes is probably outside the scope of these docs so I'd be tempted to not mention any package managers instead.

File: www/src/pages/docs/index.js [hunk 4]
```diff
@@ -22,6 +22,10 @@ class IndexRoute extends React.Component {
               Get started
             </h1>
             <p>Gatsby is a blazing fast modern site generator for React.</p>
+      <p> This quick start is intended for intermediate to advanced developers. For a gentler intro to Gatsby, <Link to="/tutorial/">head to our tutorial!</Link> </p>
+           <h2>Prerequisites</h2>
+             <p> <strong> Node.js</strong>: Before installing Gatsby, make sure you have Node.js and npm package manager installed on your system. If not, go to <a href="https://nodejs.org/" target="_blank">nodejs.org</a> and install a recent version for your operating system. </p>
```
Comment (mikelax) [inline www/src/pages/docs/index.js]: @surjithctly nice ! Can you also add `rel="noopener"` to the link on this line as well?

Comment (surjithctly) [reply www/src/pages/docs/index.js]: Yes. Done 👍

**Comment by surjithctly** [comment]
@KyleAMathews @LeKoArts Not sure why there is no space after comma even though the space is there in the code. 
screenshot

**Pull request merged**